sql_generators/active_users_aggregates/templates/desktop_query.sql

```diff
@@ -0,0 +1,107 @@
+--- Query generated via sql_generators.active_users_aggregates.
+WITH todays_metrics AS (
+  SELECT
+    client_id,
+    activity_segments_v1 AS segment,
+    'Firefox Desktop' AS app_name,
+    app_version AS app_version,
+    normalized_channel AS channel,
+    IFNULL(country, '??') country,
+    city,
+    locale,
+    first_seen_date,
+    EXTRACT(YEAR FROM first_seen_date) AS first_seen_year,
+    days_since_seen,
+    os,
+    normalized_os_version AS os_version,
+    COALESCE(
+      CAST(NULLIF(SPLIT(normalized_os_version, ".")[SAFE_OFFSET(0)], "") AS INTEGER),
+      0
+    ) AS os_version_major,
+    COALESCE(
+      CAST(NULLIF(SPLIT(normalized_os_version, ".")[SAFE_OFFSET(1)], "") AS INTEGER),
+      0
+    ) AS os_version_minor,
+    submission_date,
+    COALESCE(
+      scalar_parent_browser_engagement_total_uri_count_normal_and_private_mode_sum,
+      scalar_parent_browser_engagement_total_uri_count_sum
+    ) AS uri_count,
+    is_default_browser,
+    distribution_id,
+    attribution.source AS attribution_source,
+    attribution.medium AS attribution_medium,
+    attribution.medium IS NOT NULL
+    OR attribution.source IS NOT NULL AS attributed,
+    ad_clicks_count_all AS ad_click,
+    search_count_organic AS organic_search_count,
+    search_count_all AS search_count,
+    search_with_ads_count_all AS search_with_ads,
+    active_hours_sum
+  FROM
```

Comment:
Would you consider including `windows_build_number` to capture window versions as discussed in [DENG-514](https://mozilla-hub.atlassian.net/browse/DENG-514)?

Reply:
I'd prefer for us to implement it in a separate PR connected to the story, so that we follow our purpose of keeping PRs small and organized.